transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -33,8 +33,8 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    static {
-        // Ensure JNI is initialized by the time this class is loaded by this time!
+    {
```

Comment:
Just for fun:  instance initializers are pretty unusual:
* they run before the constructor
* they cannot explicitly throw exceptions, unlike static initializers
* there can be multiple of them, which means there are several methods named "<init>" in the class file.

Like @johnou said, they are also useful in anonymous class.  They are also what power [Double Brace initializers](http://wiki.c2.com/?DoubleBraceInitialization).